### PR TITLE
fix(ua): Update the User-Agent to use the new format

### PR DESCRIPTION
**Pull request by robertmogos**
**Summary**
Update the UA from `instantsearch.js ${version}` to `instantsearch.js (${version})`

### CODE REVIEW

**Comment by francoischalifour** [comment]
Can we add a test for this?

An appropriate place could be [`InstantSearch-test.js`](https://github.com/algolia/instantsearch.js/blob/5f41fc6ae8d03c9ddd76dff5e54e73fecc395900/src/lib/__tests__/InstantSearch-test.js#L138).

**Comment by robertmogos** [comment]
> Can we add a test for this?
> 
> An appropriate place could be [`InstantSearch-test.js`](https://github.com/algolia/instantsearch.js/blob/5f41fc6ae8d03c9ddd76dff5e54e73fecc395900/src/lib/__tests__/InstantSearch-test.js#L138).

Yes, it is on my todo list. I might need some help with that

**Comment by francoischalifour** [comment]
You can see in the test that [we mocked the JavaScript client](https://github.com/algolia/instantsearch.js/blob/5f41fc6ae8d03c9ddd76dff5e54e73fecc395900/src/lib/__tests__/InstantSearch-test.js#L148-L154), with only a `search` method for now.

The official JavaScript client also exposes an `addAlgoliaAgent` method. You can therefore add this method to the client to mock the `addAlgoliaAgent` function with [`jest.fn()`](https://jestjs.io/docs/en/jest-object#jestfnimplementation) and check that it has been called correctly at instantiation.

Let me know if you need further help.

File: src/lib/__tests__/InstantSearch-test.js [hunk 1]
```diff
@@ -2,6 +2,7 @@ import range from 'lodash/range';
 import times from 'lodash/times';
 import algoliaSearchHelper from 'algoliasearch-helper';
 import InstantSearch from '../InstantSearch';
+import version from '../version'
```
Comment (francoischalifour) [inline src/lib/__tests__/InstantSearch-test.js]: The file is not formatted: missing `;`.

File: src/lib/__tests__/InstantSearch-test.js [hunk 2]
```diff
@@ -151,6 +152,7 @@ describe('InstantSearch lifecycle', () => {
           results: requests.map(() => ({})),
         });
       },
+      addAlgoliaAgent: jest.fn()
```
Comment (francoischalifour) [inline src/lib/__tests__/InstantSearch-test.js]: The file is not formatted: missing `,`.

Comment (robertmogos) [reply src/lib/__tests__/InstantSearch-test.js]: 🤦‍♂️

File: src/lib/__tests__/InstantSearch-test.js [hunk 3]
```diff
@@ -182,6 +185,11 @@ describe('InstantSearch lifecycle', () => {
     expect(algoliasearch).toHaveBeenCalledWith(appId, apiKey);
   });
 
+  it('calls addAlgoliaAgent', () => {
+    expect(client.addAlgoliaAgent).toHaveBeenCalledTimes(1);
+    expect(client.addAlgoliaAgent).toHaveBeenCalledWith(`instantsearch.js (${version})`);
```
Comment (francoischalifour) [inline src/lib/__tests__/InstantSearch-test.js]: This line also needs to be formatted. Running `yarn lint --fix` should fix this kind of problem.

If you use Vscode, you can toggle [format on save](https://marketplace.visualstudio.com/items?itemName=cbrevik.toggle-format-on-save).

**Pull request merged**